src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -154,6 +158,69 @@ internal override string[] GetRequestCertificateAuthorities(SafeDeleteContext se
             return issuers;
         }
 
+        //
```

Comment:
This was just moved to a different file and is identical to the original, right?